### PR TITLE
Types for code v4.0.0

### PR DESCRIPTION
**Pull request by prashaantt**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header.

### CODE REVIEW

**Comment by prashaantt** [comment]
This build appears to have failed for no fault in this module

```bash
\home\travis\build\DefinitelyTyped\DefinitelyTyped\chocolatechipjs\index.d.ts
	 Line 1363: All declarations of 'responseURL' must have identical modifiers.
```

How exactly should I be dealing with this?

**Comment by vvakame** [comment]
um? could you try to merge types-2.0/HEAD?

File: code/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,189 @@
+// Type definitions for code 4.0.0
+// Project: https://github.com/hapijs/code
+// Definitions by: Prashant Tiwari <https://github.com/prashaantt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "code" {
```
Comment (vvakame) [inline code/index.d.ts]: could you strip `declare module "code" {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred

**Comment by prashaantt** [comment]
@vvakame Just want to bring to your notice that this failed again for the 3rd time. AFAICT, all builds against the `types-2.0` branch [have been failing](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/pull_requests) at least for the last one day for the same issue with `chocolatechipjs`. They are also taking unusually longer (well over an hour in most cases), while the builds against `master` complete in a couple minutes.

**Comment by DenisCarriere** [comment]
Same issues are happening with all the PR's.

**Comment by vvakame** [comment]
wow, I'm so sorry. it is our fault. Your PR is 👍

**Pull request merged**